configure.ac

```diff
@@ -74,6 +76,16 @@ AC_ARG_ENABLE([completion-polling],
     [AC_HELP_STRING([--enable-completion-polling],
                     [Enable polling in quiet, fence, and local completion operations (default:disabled)])])
 
+AC_ARG_ENABLE([hwloc-enforce-single-socket],
+    [AC_HELP_STRING([--enable-hwloc-enforce-single-socket],
+                    [Repin process if assigned affinity to cores across multiple sockets (default:disabled)])])
+AS_IF([test "$enable_hwloc_enforce_single_socket" = "yes"], [AC_DEFINE([HWLOC_ENFORCE_SINGLE_SOCKET], [1], [Re-assign CPU affinity on processes with affinity to cores on multiple sockets])])
```

Comment:
Seems like hwloc moved from the term "socket" to "package" a while back.  Are we sure we want "socket"?

(I'm good with it...)

Reply:
That's a good question. Personally, I think "socket" might be more intuitive and I may have a slight preference toward it, but I understand it's not exactly consistent with the terminology being used by hwloc